### ISSUE TITLE
Add Trace/Metric collection via Envoy Proxy

### ISSUE THREAD

**Issue by lewislabs** [lewislab-oss/appmesh-playground#1]
Can we add support for OpenTracing/OpenCensus collection from the aws envoy sidecar. 

Currently envoy doesn't directly support OpenTelemetry - https://github.com/envoyproxy/envoy/issues/9958

**Comment by janaka** [comment]
Yes, we can. I've got it working in a sandbox setup locally. Zipkin protocol -> OTel agent.

                                                                    local logs
                                                                  /
Envoy -> OTel (core) Agent -> OTel contribs collector -- Zipkin local instance
                                                                  \ 
                                                                    Honeycomb
